packages/acceptance-tests/pkg-tests-core/sources/utils/tests.ts

```diff
@@ -732,7 +732,7 @@ export const generatePkgDriver = ({
             await Promise.race([
               new Promise(resolve => {
                 // Resolve 1s ahead of the jest timeout
-                setTimeout(resolve, TEST_TIMEOUT - 1000);
+                setTimeout(resolve, TEST_TIMEOUT - 1000).unref()
```

Comment:
While this most likely works could you instead change this to clear the timeout when the promise resolves?

Reply:
Sure. Let me adjust it